### ISSUE TITLE
Apostrophe in comments messes up code coloring

### ISSUE THREAD

**Issue by DevinDow** [flutter/devtools#6360]
I have a /// comment that has an appostrophe-s in it (ex. "Task's").  
In Flutter Dev Tools Debugger, all code from that appostrophe through the end of the File is orange instead of color-coded.

___
**DevTools version**: 2.25.0
**Connected Device**:
CPU / OS: x64 (64 bit) windows
Dart Version: 3.1.1
Flutter Version: 3.13.3 / stable
Framework / Engine: 2524052335 / b8d35810e9
Connected app type: Flutter native (debug build)

**Comment by CoderDake** [comment]
@DevinDow I am not able to reproduce this with just `Task's`. Could you please include some minimum reproducible file contents that reproduce this for you?

**Comment by DevinDow** [comment]
@CoderDake It is no longer happening for me.  I opened VS Code and found that comment, and it all looks good now, and I can't make it happen.
I can't explain why not...  It happened repeatedly, and I tried lots of experimenting before opening this Issue.
And I appear to still be on the same version of everything:
Flutter 3.13.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 2524052335 (6 weeks ago) • 2023-09-06 14:32:31 -0700
Engine • revision b8d35810e9
Tools • Dart 3.1.1 • DevTools 2.25.0

Thanks for investigating & responding.  Closing this Issue.
